### ISSUE TITLE
Cyberchef hangs on JSON Beautify (and maybe others)

### ISSUE THREAD

**Issue by philhagen** [aol/moloch#1339]
**Elasticsearch version**: 7.5 (also exhibited with 7.4.x

**Moloch version**: 2.1.1

**OS name and version**: CentOS 7

**How was Moloch built/installed:** RPM

**Provide logs, stack traces and steps to reproduce**:
From browser JS Console:
```
8166f25f-6e25-4c1c-8e03-54a1d9d0dc9d:209 Refused to execute script from 'http://192.168.75.224:8005/modules/Code.js' because its MIME type ('text/html') is not executable.
Uncaught (in promise) DOMException: Failed to execute 'importScripts' on 'WorkerGlobalScope': The script at 'http://192.168.75.224:8005/modules/Code.js' failed to load.
    at blob:http://192.168.75.224:8005/8166f25f-6e25-4c1c-8e03-54a1d9d0dc9d:209:2370
    at Array.forEach (<anonymous>)
    at self.loadRequiredModules (blob:http://192.168.75.224:8005/8166f25f-6e25-4c1c-8e03-54a1d9d0dc9d:209:2211)
    at blob:http://192.168.75.224:8005/8166f25f-6e25-4c1c-8e03-54a1d9d0dc9d:209:432
    at u (blob:http://192.168.75.224:8005/8166f25f-6e25-4c1c-8e03-54a1d9d0dc9d:209:3924)
    at Generator._invoke (blob:http://192.168.75.224:8005/8166f25f-6e25-4c1c-8e03-54a1d9d0dc9d:209:3712)
    at Generator.A.forEach.e.<computed> [as next] (blob:http://192.168.75.224:8005/8166f25f-6e25-4c1c-8e03-54a1d9d0dc9d:209:4347)
    at u (blob:http://192.168.75.224:8005/8166f25f-6e25-4c1c-8e03-54a1d9d0dc9d:209:3924)
    at t (blob:http://192.168.75.224:8005/8166f25f-6e25-4c1c-8e03-54a1d9d0dc9d:209:4485)
    at blob:http://192.168.75.224:8005/8166f25f-6e25-4c1c-8e03-54a1d9d0dc9d:209:4789
```

When selecting the "JSON Beautify" module, the CyberChef instance hangs, showing only the processing animation.  I'm not sure if this happens with others (I've tested a few handfuls and it seems to be common with several JSON-related modules, but that's not exhaustive.)

**Comment by philhagen** [comment]
I should also add that pasting the same hex bytes into a standalone CyberChef instance performs as expected, with the decode (or error) coming up immediately. (Tested with 9.11.5 and 9.11.14)

**Comment by awick** [comment]
fixed in 2.1.2
